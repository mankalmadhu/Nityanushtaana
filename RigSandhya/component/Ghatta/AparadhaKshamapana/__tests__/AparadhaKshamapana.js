import React from 'react';
import ಅಪರಾಧಕ್ಷಮಾಪಣ from '../AparadhaKshamapana';
import Renderer from 'react-test-renderer';

jest.mock('../../Ghatta.vinyasa');
jest.mock('../../../Vishaya/Vishaya',()=>'ವಿಷಯ');
jest.mock('../../../Vishaya/Viniyoga',()=>'ವಿನಿಯೋಗ');
jest.mock('../../../Vishaya/Purvanga',()=>'ಪೂರ್ವಾ೦ಗ');
jest.mock('../../../Vishaya/Sheershike',()=>'ಶೀರ್ಷಿಕೆ');

test("ಅಪರಾಧಕ್ಷಮಾಪಣ", () => {
   expect(Renderer.create(
    <ಅಪರಾಧಕ್ಷಮಾಪಣ />
  ).toJSON()).toMatchSnapshot();

});