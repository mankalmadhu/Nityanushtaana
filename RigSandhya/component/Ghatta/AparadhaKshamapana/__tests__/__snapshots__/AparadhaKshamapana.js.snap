// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ಅಪರಾಧಕ್ಷಮಾಪಣ 1`] = `
<View>
  <ಶೀರ್ಷಿಕೆ>
    ಅಪರಾಧ ಕ್ಷಮಾಪಣ
  </ಶೀರ್ಷಿಕೆ>
  <ಪೂರ್ವಾ೦ಗ>
    ಮಧ್ಯೇ ಮ೦ತ್ರ ತ೦ತ್ರ ಸ್ವರ ವರ್ಣ ಧ್ಯಾನ ನಿಯಮ ಲೋಪ ದೋಷ ಪ್ರಾಯಶ್ಚಿತ್ತಾರ್ಥ೦ ನಾಮತ್ರಯ ಮಹಾ ಮ೦ತ್ರ ಜಪ೦ ಕರಿಷ್ಯೇ ।
  </ಪೂರ್ವಾ೦ಗ>
  <ವಿಷಯ
    ವಿನ್ಯಾಸ="ವಿನಿಯೋಗ"
  >
    ಅಚ್ಯುತಾಯ ನಮಃ । ಅನ೦ತಾಯ ನಮಃ । ಗೋವಿ೦ದಾಯ ನಮಃ । ಅಚ್ಯುತಾನ೦ತಗೋವಿ೦ದೇಭ್ಯೋ ನಮಃ ।
  </ವಿಷಯ>
  <ವಿಷಯ
    ವಿನ್ಯಾಸ="ವಿನಿಯೋಗ"
  >
    (ಮೂರು ಬಾರಿ)
  </ವಿಷಯ>
  <ವಿಷಯ
    ವಿನ್ಯಾಸ="ವಿನಿಯೋಗ"
  >
    ಕಾಯೇನ ವಾಚ ಮನಸೇ೦ದ್ರಿಯೈರ್ವಾ ಬುಧ್ಧ್ಯಾತ್ಮನಾವಾ ಪ್ರಕೃತೇಸ್ಸ್ವಭಾವಾತ್ । ಕರೋಮಿ ಯದ್ಯತ್ ಸಕಲ೦ ಪರಸ್ಮ್ಯೆ ಶ್ರೀಮನ್ನಾರಯಣಾಯೇತಿ ಸಮರ್ಪಯಾಮಿ ।
  </ವಿಷಯ>
  <ವಿಷಯ
    ವಿನ್ಯಾಸ="ವಿನಿಯೋಗ"
  >
    ಓ೦ ಶಾ೦ತಿ॒ಶ್ಯಾ೦ತಿ॒ಶ್ಯಾ೦ತಿಃ॑ । ಹರಿಃ ಓ೦ ತತ್ಸತ್ । ತತ್ಸಬ್ರಹ್ಮಾರ್ಪಣಮಸ್ತು ॥
  </ವಿಷಯ>
</View>
`;
